<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>directory-sizes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_root_dir">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDockWindow">
    <property name="title">
     <string>Окна</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuDockWindow"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarGeneral">
   <property name="windowTitle">
    <string>General</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_go"/>
   <addaction name="action_show_in_explorer"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSettings">
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Dir:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="line_edit_dir_path">
           <property name="text">
            <string>C:\</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_select_dir">
           <property name="toolTip">
            <string>Select dir</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_min_size">
         <property name="text">
          <string>Filter size:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_edit_filter_size">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Пример: {size} &amp;gt;= %1GB% and {size} &amp;lt;= %3GB%&lt;/p&gt;&lt;p&gt;size: размер папок в байтах, который будет подставляться в {size}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>{size} &gt;= %1GB%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLog">
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_go">
   <property name="text">
    <string>Go</string>
   </property>
   <property name="toolTip">
    <string>Go</string>
   </property>
  </action>
  <action name="action_show_in_explorer">
   <property name="text">
    <string>Show in Explorer</string>
   </property>
   <property name="toolTip">
    <string>Show in Explorer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>line_edit_dir_path</sender>
   <signal>textChanged(QString)</signal>
   <receiver>label_root_dir</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
